.github/dependabot.yml

```diff
@@ -5,31 +5,42 @@
 # these updates when the pull request(s) in the appropriate skeleton are merged
 # and Lineage processes these changes.
 
-version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
+  - directory: /
     ignore:
       # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
       - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
+      - dependency-name: crazy-max/ghaction-dump-context
+      - dependency-name: crazy-max/ghaction-github-labeler
+      - dependency-name: crazy-max/ghaction-github-status
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
-
-  - package-ecosystem: "pip"
-    directory: "/"
+      - dependency-name: step-security/harden-runner
+    package-ecosystem: github-actions
     schedule:
-      interval: "weekly"
+      interval: weekly
+
+  - directory: /
+<<<<<<< HEAD
     ignore:
       # Managed by cisagov/skeleton-ansible-role
-      - dependency-name: "ansible"
-      - dependency-name: "ansible-lint"
+      - dependency-name: ansible
+      - dependency-name: ansible-lint
+=======
+    # ignore:
+    #   # Managed by cisagov/skeleton-ansible-role
+    #   - dependency-name: ansible
+    #   - dependency-name: ansible-lint
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
```

Comment:
@jsf9k - This is definitely a problem - merge markers in `dependabot.yml`.

Reply:
Something got hosed when I rebased.  I just redid the upstream merge and force pushed, so please re-review.